CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- `rel` prop passed down in the `return` statement
```

Comment:
This is how you solved, but what is the problem? Please describe the bug or unexpected behaviour

Reply:
Hi! Sure, I'm sorry I didn't explained it before.
So, I created a WS with the actual state:
[ws](https://pr620before--ruwhirlpool.myvtex.com/)
at this [link](https://pr620before--ruwhirlpool.myvtex.com/sushilnaya-mashina-whirlpoolpro-awz8hps-869991580330/p), below the green button there is another one that should have an `<a>` tag with a `rel` attribute of `"nofollow"` but it doesn't work (based on the work of a PR that was recently approved, #618)
This PR should make the rel attribute appear in the Link component where this prop is specified.

![immagine](https://user-images.githubusercontent.com/35635227/130426986-39fc6682-db9d-4618-b953-fc2dc4568f02.png)

In this [workspace](https://martinolisprint10--ruwhirlpool.myvtex.com/sushilnaya-mashina-whirlpoolpro-awz8hps-869991580330/p), with the modifications I'm discussing with @igorbrasileiro, the `<a>` tag has the `rel` attribute as intended.
Let me know if you need any other explanation :)

Reply:
```suggestion
-  `rel` prop received by `Link` component not being added to the rendered `<a>` tags.
```